### ISSUE TITLE
Not working on 2.2.2 - "Error stopping node: Close timed out"

### ISSUE THREAD

**Issue by ozonejunkieau** [windkh/node-red-contrib-telegrambot#247]
I'm having some issues getting this setup on my NodeRed deployment.

My installation is running via Podman using the 2.2.2 standard image from dockerhub, with a persistent data directory.

After following the instructions for configuration, I get... nothing?

* The configuration seems to work, and I have no issues following the configuration. If I make any change to the configuration, then do a full redeploy all, I get a `"Error stopping node: Close timed out"` in the logs. The node it refers to is the main bot config.
* The tokens work, I have confirmed this by directly accessing the API via a browser.
* I don't believe the polling is ever working as the updates are never "removed" from the list accessible in the browser.
* Connecting to the container and issuing a curl command to `https://api.telegram.org/bot<token>/getUpdates` gives me the correct response, so I believe connectivity should be fine.
* I have enabled trace level debugging on NodeRed and accessing the logs I don't see any errors associated with the bot at all. In fact, the only log lines it generates are:
```
24 May 17:35:56 - [trace] Stopping node telegram bot:b3275cb5fcd52144
24 May 17:36:11 - [error] [telegram bot:b3275cb5fcd52144] Error stopping node: Close timed out
24 May 17:36:11 - [trace] [flow:global]  b3275cb5fcd52144 | telegram bot | 
```

For what it's worth, I also have the following installed:
```
        "node-red-contrib-avr-yamaha": "^0.8.9",
        "node-red-contrib-hysteresis": "~0.3.0",
        "node-red-contrib-influxdb": "~0.6.1",
        "node-red-contrib-node-lifx": "~0.10.1",
        "node-red-contrib-samsung-tv-control": "~1.3.15",
        "node-red-contrib-tasmota": "~0.9.12",
        "node-red-contrib-telegrambot": "~11.3.0",
        "node-red-contrib-tplink": "~1.0.0"

```
None of these seem likely to be an issue, but I'm at my wits end a bit here. Any suggestions on how to debug this further would be greatly appreciated. Thanks in advance.

**Comment by windkh** [comment]
@ozonejunkieau hi, I guess the code that polls the TG server in hanging somehow. When redeploying the following like does not return from stopPolling.
self.telegramBot.stopPolling().then(setStatusDisconnected);

Hm normally this would indicate a network problem.... I guess that I can not really provide a solution from what I see from here.

**Comment by grahamwoan** [comment]
On start-up, the node-red log says:

> ###
> ### The "request" library is not installed automatically anymore.
> ### But is a dependency of "request-promise".
> ### Please install it with:
> ### npm install request --save
> ###

and I had to install `request` manually to get telegrambot working again.

**Comment by windkh** [comment]
should be fixed in 11.5.0
